Bottom: a0f1c62c4941d94ba2453eb0d02217a23a3a6f05
Top:    6708b79de592d1a3ab6952c863efa5d932faaf9f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-24 18:58:20 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
